source/extensions/filters/http/rbac/rbac_filter.cc

```diff
@@ -65,13 +65,30 @@ RoleBasedAccessControlRouteSpecificFilterConfig::RoleBasedAccessControlRouteSpec
 
 Http::FilterHeadersStatus RoleBasedAccessControlFilter::decodeHeaders(Http::HeaderMap& headers,
                                                                       bool) {
+  ENVOY_LOG(
+      debug,
+      "checking request:\n"
```

Comment:
"checking request" isn't a very descriptive message. Either make a new "rbac" log type, or prefix all of the messages you added with "rbac: " (or something similar).

Reply:
Thanks for the suggestion, I added a new "rbac" log type.